implementations/rust/ockam/ockam_command/src/tcp/outlet/list.rs

```diff
@@ -73,7 +73,23 @@ async fn run_impl(
         &format!("Outlets on Node {node_name}"),
         &format!("No TCP Outlets found on node {node_name}."),
     )?;
-    opts.terminal.stdout().plain(list).write_line()?;
+    let json: Vec<_> = outlets
+        .list
+        .iter()
+        .map(|outlet| {
+            Ok(serde_json::json!({
+                "name": outlet.alias,
+                "from": outlet.worker_address()?,
+                "to": outlet.socket_addr,
+            }))
+        })
+        .flat_map(|res: Result<_, ockam_core::Error>| res.ok())
+        .collect();
+    opts.terminal
+        .stdout()
+        .plain(list)
+        .json(serde_json::json!(json))
```

Comment:
I think you should be able to just run `serde_json::json!(&outlets.list)`

Reply:
This is what that outputs; looks like there are a few extra fields. Which do you & the team prefer?

![image](https://github.com/build-trust/ockam/assets/772507/c753157f-445b-4475-9855-dc9edf8586b1)

Reply:
You are right, your approach is just what we need, thanks! 🙏